algorithms/C/sorting/radix-sort.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+
+
+int get_max( int arr[], int n){
+
+    int myMax = -1 ;
+    for(int i=0; i<n; i++){
+        if(arr[i] > myMax)myMax = arr[i] ;
+    }
+
+    return myMax ;
+}
+
+void count_sort(int arr[], int n, int digit){
+    int count[10] = {0} ;
+    int sorted[n] ;
```

Comment:
I would not recommend using a Variable Length Array (VLA) here.
Although C99 and above C standards accept it, it is still considered as a code smell and can cause troubles when used with large value of `n`.

A heap allocated array should do the trick here.

Reply:
But all resources use the same idea that I'm used above !? 
Anyway could you please tell me what'd U recommend `calloc()` or `malloc()` ?